scripts/ignore-openapi

```diff
@@ -11,18 +13,20 @@ AFTER_COMMIT_ID=${DRONE_COMMIT}
 FILES=$(git --no-pager diff --name-only ${BEFORE_COMMIT_ID}...${AFTER_COMMIT_ID})
 
 # if the api/swaggers.json has changed
-if echo "$FILES" | grep -q "api/swagger.json"; then
+if echo "$FILES" | grep -qE "v(.*)swagger.json"; then
     # perform re-generate and check for dirty files
-    yarn clean-api-docs all
-    yarn gen-api-docs all
-    dirty_files=$(git status --porcelain --untracked-files=no)
+    yarn clean-api-docs all && yarn gen-api-docs all
+    yarn docusaurus clean-api-docs:version api:all && yarn docusaurus gen-api-docs:version api:all
```

Comment:
is this redundant? does version also includes the default one?

Reply:
Removed redundant command line.